learn/advanced/updating.md

```diff
@@ -201,59 +147,222 @@ The server should return a response that looks like this:
 }
 ```
 
-This command returns a `taskUid`. You can use this to [track the status](/reference/api/tasks.md#get-one-task) of your dump. Keep in mind that the process can take some time to complete.
+Use the `taskUid` to [track the status](/reference/api/tasks.md#get-one-task) of your dump. Keep in mind that the process can take some time to complete.
 
-Once the `dumpCreation` task shows `"status": "succeeded"`, you're ready to move on.
+::: note
+The response will vary slightly depending on your version. For v0.27 and below, the response returns a dump `uid`. You can track the status of the dump using the get dumps status endpoint:
 
-```json
-{
-  "uid": 1,
-  "indexUid": null,
-  "status": "succeeded",
-  "type": "dumpCreation",
-  "canceledBy": null,
-  "details": {
-    "dumpUid": "20220621-161029217"
-  },
-  "error": null,
-  "duration": "PT0.025872S",
-  "enqueuedAt": "2022-06-21T16:10:29.217688Z",
-  "startedAt": "2022-06-21T16:10:29.218297Z",
-  "finishedAt": "2022-06-21T16:10:29.244169Z"
-}
+```sh
+  curl \
+    -X GET 'http://<your-domain-name>/dumps/:dump_uid/status'
+    -H 'Authorization: Bearer API_KEY' 
+  # -H 'X-Meili-API-Key: API_KEY' for v0.24 or below
 ```
 
-## Step 4: Stop the Meilisearch instance
+:::
+
+Once the `dumpCreation` task shows `"status": "succeeded"`, you're ready to move on.
 
-Stop your Meilisearch instance. If you're running Meilisearch as a `systemctl` service, you can stop your instance with:
+## Step 2: Prepare for migration
+
+### Stop the Meilisearch instance
+
+Stop your Meilisearch instance.
+
+:::: tabs
+
+::: tab Local installation
+
+If you're running Meilisearch locally, you can stop the program with `Ctrl + c`.
+
+:::
+
+::: tab Cloud platforms
+
+If you're running Meilisearch as a `systemctl` service, connect via SSH to your cloud instance and execute the following command to stop Meilisearch:
 
 ```bash
 systemctl stop meilisearch
 ```
 
-## Step 5: Delete the database folder
+You may need to prefix the above command with `sudo` if you are not connected as root.
 
-To delete the old Meilisearch version, you need to delete the `data.ms` folder. `data.ms` should be at the root of the Meilisearch binary, unless you chose [another location](/learn/configuration/instance_options.md#database-path).
+:::
+
+::::
+
+### Create a backup
+
+Instead of deleting `data.ms`, we suggest creating a backup in case something goes wrong. `data.ms` should be at the root of the Meilisearch binary, unless you chose [another location](/learn/configuration/instance_options.md#database-path).
+
+On **cloud platforms**, you will find the `data.ms` folder at `/var/lib/meilisearch/data.ms`.
+
+Move the binary of the current Meilisearch installation and database to the `/tmp` folder:
+
+:::: tabs
+
+::: tab Local installation
+
+```
+mv /path/to/your/meilisearch/directory/meilisearch /tmp
+mv /path/to/your/meilisearch/directory/meilisearch/data.ms /tmp/
```

Comment:
I could be wrong, but I think if you try to execute these commands in order they will not work well, since the first command moves the `meilisearch` folder, and the `data.ms` folder is located within it. It probably makes more sense to move `data.ms` first, then move `meilisearch`. Or just use a single command to move `meilisearch` and keep `data.ms` within it. Right? 🤔 

**This comment does not apply to cloud platforms**, because in that case they are actually located in two different places.

Reply:
The second command `mv /path/to/your/meilisearch/directory/meilisearch/data.ms /tmp/` should be `mv /path/to/your/meilisearch/directory/data.ms /tmp/`. The first one is moving the binary, not the entire folder